src/NULS/Address.h

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#pragma once
+#include "../PrivateKey.h"
+#include "../PublicKey.h"
+
+#include <string>
+
+namespace TW::NULS {
+
+class Address {
```

Comment:
now we have a base `Base58Address`, could you please inherit `Address` from it? less code then